src/main/java/pro/sky/collections1/service/EmployeeServiceImpl.java

```diff
@@ -0,0 +1,48 @@
+package pro.sky.collections1.service;
+
+import org.springframework.stereotype.Service;
+import pro.sky.collections1.Employee;
+import pro.sky.collections1.NotFoundException;
+import pro.sky.collections1.OverflowArrayException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class EmployeeServiceImpl implements EmployeeService {
+
+    List employee = new ArrayList();
```

Comment:
List является параметризованным классом, ты можешь указать какие значения будут храниться в этой коллекции. Это нужно чтобы не допустить попадания в коллекцию объектов другого типа. 
```suggestion
    List<Employee> employee = new ArrayList<>();
```